webapp/src/widgets/propertyMenu.tsx

```diff
@@ -68,10 +67,10 @@ const PropertyMenu = React.memo((props: Props) => {
                 onClick={(e) => e.stopPropagation()}
                 onChange={(e) => setName(e.target.value)}
                 value={name}
-                onBlur={() => props.onNameChanged(name)}
+                onBlur={() => props.onTypeAndNameChanged(undefined, name)}
```

Comment:
Again, here, I prefer to not use the default value, looks weird and hard to undrestand and follow, and you already have the propertyType in the properties.

Reply:
Done! Please refresh